jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -191,8 +188,8 @@ class PlotOptions(PluginTemplateMixin):
     stretch_vmax_sync = Dict().tag(sync=True)
 
     stretch_hist_zoom_limits = Bool().tag(sync=True)
-    stretch_hist_nbins = IntHandleEmpty().tag(sync=True)
-    stretch_histogram = Any().tag(sync=True, **widget_serialization)
+    stretch_hist_nbins = IntHandleEmpty(25).tag(sync=True)

```

Comment:
Where did 25 come from?

Reply:
The previous implementation used 25 bins by default in https://github.com/spacetelescope/jdaviz/pull/2314/files#diff-522ee60e0cfb6e1041a540ed4c2a07cd8efc622ca39f31bd55af884d0cc4d96dL590